pkg/common/utils/utils.go

```diff
@@ -104,12 +104,21 @@ func IsAssignedPod(pod *v1.Pod) bool {
 }
 
 func GetQueueNameFromPod(pod *v1.Pod) string {
+	// Queue name can be defined in multiple places
+	// The queue name is determined by the following order
+	// 1. Label: constants.CanonicalLabelQueueName
+	// 2. Annotation: constants.AnnotationQueueName
+	// 3. Label: constants.LabelQueueName
+	// 4. Default: constants.ApplicationDefaultQueue
```

Comment:
Note:  the order to fetch queue name from pod is:

Before this PR,
1. Label: constants.LabelQueueName
2. Annotation: constants.AnnotationQueueName
3. Default: constants.ApplicationDefaultQueue

After this PR
1. **Label: constants.CanonicalLabelQueueName (New)**
2. Annotation: constants.AnnotationQueueName
3. Label: constants.LabelQueueName (**Previous: constants.LabelQueueName > constants.AnnotationQueueName**)
4. Default: constants.ApplicationDefaultQueue